patches/components-download-internal-background_service-proto_conversions.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/components/download/internal/background_service/proto_conversions.cc b/components/download/internal/background_service/proto_conversions.cc
+index 560fd484d0362466d9b1af423fe319156441adf1..dca3b9a1b6f4167f6a973cbda9ef02cb4c6b3b81 100644
+--- a/components/download/internal/background_service/proto_conversions.cc
++++ b/components/download/internal/background_service/proto_conversions.cc
+@@ -76,6 +76,8 @@ protodb::DownloadClient ProtoConversions::DownloadClientToProto(
+       return protodb::DownloadClient::OPTIMIZATION_GUIDE_PREDICTION_MODELS;
+     case DownloadClient::BOUNDARY:
+       return protodb::DownloadClient::BOUNDARY;
++    case DownloadClient::CUSTOM_LIST_SUBSCRIPTIONS:
```

Comment:
please follow patch guidelines https://github.com/brave/brave-browser/wiki/Patching-Chromium

Reply:
I will fix these patches; could you also take a look at https://github.com/brave/brave-core/pull/8124#discussion_r627065579?